### PR TITLE
bottles/similar

### PR DESCRIPTION
**Pull request by OceanUwU**

### CODE REVIEW

File: steam_workshop/config.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "steamPublishedID": "2843056085",
-  "title": "Downfall Relic Stats compatibility",
-  "description": "Adds Relic Stats compatibility to the Downfall Expansion.",
+  "title": "Downfall Relic Stats",
+  "description": "",
```
Comment (iTitus) [inline steam_workshop/config.json]: please undo this change

File: src/main/java/io/github/ititus/downfallRelicStats/DownfallRelicStats.java [hunk 2]
```diff
@@ -51,5 +61,14 @@ public void receivePostInitialize() {
...
         register(BlackPowderInfo.getInstance());
+        RelicStats.registerCustomStats(BottledAnomaly.ID, new BottledAnomalyInfo());
+        RelicStats.registerCustomStats(BottledStasis.ID, new BottledStasisInfo());
+        RelicStats.registerCustomStats(BottledCode.ID, new BottledCodeInfo());
+        RelicStats.registerCustomStats(SackOfGems.ID, new SackOfGemsInfo());
+        RelicStats.registerCustomStats(GemstoneGun.ID, new GemstoneGunInfo());
+        RelicStats.registerCustomStats(D8.ID, new D8Info());
+        RelicStats.registerCustomStats(GremlinSack.ID, new GremlinSackInfo());
+        RelicStats.registerCustomStats(SignatureFinisher.ID, new SignatureFinisherInfo());
+        RelicStats.registerCustomStats(StasisEgg.ID, new StasisEggInfo());
```
Comment (iTitus) [inline src/main/java/io/github/ititus/downfallRelicStats/DownfallRelicStats.java]: Please use `register(....)` here

**Review by iTitus — CHANGES_REQUESTED** [review]
Please fix merge conflicts and make your stats classes inherit from `BaseRelicStats`. This requires some refactors. Just look at my other relic stats classes, they should give you an idea how it might look.

File: steam_workshop/config.json (hunk 1 above)
Comment (OceanUwU) [reply steam_workshop/config.json]: ive put the title back, but its basically never worth putting anything in the description since itll just override whatever u do in steam
